ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -279,24 +289,101 @@ def generate_arp_responder_conf(self, vlan_host_map):
 
     def dump_arp_responder_config(self, dump):
         # save data for arp_replay process
-        with open("/tmp/from_t1.json", "w") as fp:
+        filename = "/tmp/from_t1.json" if self.preboot_oper is None else "/tmp/from_t1_%s.json" % self.preboot_oper
+        with open(filename, "w") as fp:
             json.dump(dump, fp)
 
+    def get_peer_dev_info(self):
+        content = self.read_json('peer_dev_info')
+        for key in content.keys():
+            if 'ARISTA' in key:
+                self.vm_dut_map[key] = dict()
+                self.vm_dut_map[key]['mgmt_addr'] = content[key]['mgmt_addr']
+                # initialize all the port mapping
+                self.vm_dut_map[key]['dut_ports'] = []
+                self.vm_dut_map[key]['neigh_ports'] = []
+                self.vm_dut_map[key]['ptf_ports'] = []
+
+    def get_portchannel_info(self):
+        content = self.read_json('portchannel_ports_file')
+        for key in content.keys():
+            for member in content[key]['members']:
+                for vm_key in self.vm_dut_map.keys():
+                    if member in self.vm_dut_map[vm_key]['dut_ports']:
+                        self.vm_dut_map[vm_key]['dut_portchannel'] = key
+                        self.vm_dut_map[vm_key]['neigh_portchannel'] = 'Port-Channel 1'
```

Comment:
Is it always `Port-Channel 1`?

Reply:
Yes. for all T0 topologies, Arista VMs have only 1 portchannel and it is port-channel1